.github/release-notes/egress-gateway-ipv6.yaml

```diff
@@ -0,0 +1,4 @@
+kind: feature
```

Comment:
I don't think we usually use yaml files for release notes AFAIK, we can probably remove this file.